subhub/api/webhooks/stripe/invoices.py

```diff
@@ -10,38 +10,36 @@
 
 class StripeInvoiceFinalized(AbstractStripeWebhookEvent):
     def run(self):
-        data = {
-            "event_id": self.payload.id,
-            "event_type": self.payload.type,
-            "invoice_id": self.payload.data.object.id,
-            "customer_id": self.payload.data.object.customer,
-            "subscription_id": self.payload.data.object.subscription,
-            "currency": self.payload.data.object.currency,
-            "charge": self.payload.data.object.charge,
-            "period_start": self.payload.data.object.period_start,
-            "period_end": self.payload.data.object.period_end,
-            "amount_paid": self.payload.data.object.amount_paid,
-            "invoice_number": self.payload.data.object.number,
-            "description": self.payload.data.object.lines.data[0].description,
-            "application_fee_amount": self.payload.data.object.application_fee_amount,
-            "created": self.payload.data.object.created,
-        }
+        data = self.create_data(
+            invoice_id=self.payload.data.object.id,
+            customer_id=self.payload.data.object.customer,
+            subscription_id=self.payload.data.object.subscription,
+            currency=self.payload.data.object.currency,
+            charge_id=self.payload.data.object.charge,
+            period_start=self.payload.data.object.period_start,
+            period_end=self.payload.data.object.period_end,
+            amount=self.payload.data.object.amount_paid,
+            invoice_number=self.payload.data.object.number,
+            description=self.payload.data.object.lines.data[0].description,
+            application_fee_amount=self.payload.data.object.application_fee_amount,
+            created=self.payload.data.object.created,
+        )
+        logger.info(f"invoice finalized {data}")
```

Comment:
fix logging

Reply:
addressed in update

Reply:
this doesn't seem to be addressed by passing `data` as a keyword argument